.mergify.yml

```
@@ -2,9 +2,9 @@ pull_request_rules:
   - name: rebase and merge when passing all checks
     conditions:
       - base=master
-      - status-success="validate commits (3.6)"
-      - status-success="validate commits (3.7)"
-      - status-success="validate commits (3.8)"
+      - status-success="validate commits"
+      - status-success="Lint with flake8"
+      - status-success="Lint with black"
```

actually I *think* the `black` and `flake8` checks will both be reported under the same status since they are serial checks run under the `run-checks:` job. If you want to split these out into separate checks then you can do so by moving each to a new "job" (at the same level as `run-checks` and `check-pr`.)

If you are fine with checks running serially, then you can just update the status name to
```suggestion
      - status-success="run-checks (3.7)"
```

---

Ah, okay. I think I'm fine with the checks just running serially. I'll go ahead and push a change to update the status name to `run checks (3.7)`.